lib/index.js

```diff
@@ -170,7 +170,7 @@ exports.unauthorized = function (message, scheme, attributes) {          // Or f
             err.output.payload.attributes = {};
         }
 
-        if (attributes) {
```

Comment:
I would still structure it by checking for attributes first, then handling the diff types.